### DISCUSSION TITLE
tool hotkey + tab to choose between tool functions

### DISCUSSION THREAD

**Discussion by age3rcm** [opentoonz/opentoonz#6561] [Ideas]
### Is your feature request related to a problem? Please describe

I bet the workflow will improve if there is a quick way to do it on the keyboard to change tool function, precisely, animation tool
even if there is the tool function where all of them in one exists, this could also be applied to select tool and geometry tool

### Describe the solution you'd like

keep pressed the tool hotkey then press tab to cycle through the other functions. the name of these functions appears next to the cursor in pink text!

**Comment by artisteacher** [comment]
You can always just change the shortcuts to use the "Tool - Next Mode", "Tool - Next Type", or "Geometric Tool - Next Shape".

Ie. changing the shortcut A to work for "Animate Tool - Next Mode" lets you press A to switch to the Animate Tool. When the Animate Tool is active, pressing A cycles through the different modes. 

I'd love for this to be the default - I don't get why it's not. 

Though I'm not sure what I'd rather have as the default for the Fill Tool since there's both "Next Type" and "Next Mode". What do people change more often - filling lines/areas or the selection type?

Btw., this was added in https://github.com/opentoonz/opentoonz/pull/3213
